backend/src/test/resources/responses/datahub-error-response.json

```diff
@@ -1,5 +1,4 @@
 {
-  "id" : "7535d2c9-eeba-4b47-9e89-bbdf4f862421",
```

Comment:
So we have an existing integration test that would have caught this issue if it hadn't had the id field - which I don't think we realized was optional at the time.  Removing the id from the mock response caused the test to fail and adding the migration caused it to pass.